Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -1,5 +1,10 @@
 use std::io;
 
+fn calc(str: String) -> i32 {
```

Comment:
Please separate the input parsing from the calculation. Input parsing is part of the interface between the program and the user, while application logic works on normalized data.

Reply:
:+1: